autoload/neomake/makers/ft/c.vim

```diff
@@ -1,11 +1,17 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#c#EnabledMakers()
+    let makers = []
```

Comment:
This can just be `let makers = ['clang-tidy']`. You only need to do the exists check yourself if there are special requirements (e.g. only enable maker x if maker y doesn't exist).